core/src/main/java/org/apache/iceberg/SortOrderParser.java

```diff
@@ -50,6 +52,26 @@ public static void toJson(SortOrder sortOrder, JsonGenerator generator) throws I
     generator.writeEndObject();
   }
 
+  public static String toJson(SortOrder sortOrder) {
+    return toJson(sortOrder, false);
+  }
+
+  public static String toJson(SortOrder sortOrder, boolean pretty) {
+    try {
+      StringWriter writer = new StringWriter();
+      JsonGenerator generator = JsonUtil.factory().createGenerator(writer);
+      if (pretty) {
+        generator.useDefaultPrettyPrinter();
+      }
+      toJson(sortOrder, generator);
+      generator.flush();
+      return writer.toString();
+
+    } catch (IOException e) {
+      throw new RuntimeIOException(e);
```

Comment:
nit: is there any reason for us to still use this deprecated exception instead of just `UncheckedIOException`?

Reply:
That was copy & paste error. I'll update it.

Reply:
Fixed.